app/assets/javascripts/comments.js

```diff
@@ -1,5 +1,31 @@
 $( function () {
-  $('.edit_comment #comment_content').live('change', RescueRails.saveParentForm);
+
+  if ( $('.edit_comment').length > 0) {
+
+    $('.toggle-edit-comment').each(function(){
```

Comment:
> 4) After posting a new comment the edit buttons don't work at all. (Any guesses on this one?

you'll want to make this line `.on('click', function() {` instead of `.each` and then you'd have to remove the isEditing and click lines.  Then instead of an isEditing var maybe store it in a data attribute somewhere?